etc/config-template.toml

```diff
@@ -11,17 +11,33 @@
 # log-file = ""
 
 [readpool.storage]
-# size of thread pool for high-priority read operations
+# size of thread pool for high-priority operations
 # high-concurrency = 8
-# size of thread pool for normal-priority read operations
+# size of thread pool for normal-priority operations
 # normal-concurrency = 8
-# size of thread pool for low-priority read operations
+# size of thread pool for low-priority operations
 # low-concurrency = 8
-# max running high-priority read operations, reject if exceed
+# max running high-priority operations, reject if exceed
 # max-tasks-high = 16000
-# max running normal-priority read operations, reject if exceed
+# max running normal-priority operations, reject if exceed
 # max-tasks-normal = 16000
-# max running low-priority read operations, reject if exceed
+# max running low-priority operations, reject if exceed
+# max-tasks-low = 16000
+# size of stack size for each thread pool
+# stack-size = "10MB"
+
+[readpool.coprocessor]
```

Comment:
Actually storage and coprocessor use the same readpool, should we need two sections here?